src/mongoose_metrics.erl

```diff
@@ -162,6 +163,15 @@ get_dist_data_stats() ->
 get_up_time() ->
     {value, erlang:round(element(1, erlang:statistics(wall_clock))/1000)}.
 
+-spec tcp_ports() -> {value, integer()}.
+tcp_ports() ->
+    {value, length(port_list(name, "tcp_inet"))}.
+
+-spec port_list(Attr::atom(), term()) -> [port()].
+port_list(Attr, Val) ->
```

Comment:
needs to be tested, how good it works with 300k ports.
Last time I've tried, this code is sloooow.

So, calling it from exometer function evaluation would delay exometer reporting, which can cause missed window in graphite.

Reply:
Now that sounds like something that worries me. I've been reading about exometer's cached probes than can run in their own processes, but I don't know how to use them, we don't have a single one of them. Any help? :(

Reply:
@arcusfelis 

What do you think about using something like `os:cmd("lsof -i -a -p " ++ os:getpid() ++ " | wc -l").`?